apps/demos/Demos/CardView/SimpleArray/Angular/app/app.component.html

```diff
@@ -0,0 +1,6 @@
+<dx-card-view id="cardView" [dataSource]="customers" keyExpr="ID">
```

Comment:
Let's format it as we do it in other Angular demo:
```suggestion
<dx-card-view 
  [dataSource]="customers"
  keyExpr="ID">
```

Also, let's remove the ```id``` attribute
https://js.devexpress.com/Angular/Demos/WidgetsGallery/Demo/DataGrid/SimpleArray/MaterialBlueLight/

Reply:
we have prettier